demos/sev-demo/README.md

```diff
@@ -0,0 +1,207 @@
+# CCv0 with AMD SEV
+
+CCv0 can be used with AMD SEV. Check out this [demo video](https://youtu.be/SXZ-hzhjahU) to see it in action. This guide explains how to take an SEV node and use CCv0 with it. This guide does not cover configuring an SEV machine. You need to have a node where you can start SEV VMs. This guide also does not cover setting up CCv0. You need a working deployment of CCv0. You will update this deployment to use SEV. 
+Soon many of these steps will be automated via [Kata Deploy](https://github.com/kata-containers/kata-containers/tree/main/tools/packaging/kata-deploy) and the [Confidential Containers Operator](https://github.com/confidential-containers/operator).
+
+## OVMF
+
+You will need to use OVMF. Furthemore, you will need to use the AMDSEV build of OVMF. An appropriate [firmware binary](OVMF.fd) is available in this directory.
```

Comment:
Seems wrong to push a binary into a documentation repository. Alternatively, we could provide all the demo binaries (ovmf, kernel, inird) in a separate `sev-demo-files` repository

Reply:
Yeah. I was a bit on the fence about putting the demos in the documentation repo at all. I think one file isn't quite reason enough to make a new repo tho.